quic/s2n-quic-core/src/dc/mod.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I've been preferring moving away from `mod.rs` and using the actual name, since it gets annoying having a million `mod.rs` files in the project.